src/main/java/com/sourcegraph/action/FileAction.java

```diff
@@ -6,10 +8,18 @@
 import com.intellij.openapi.fileEditor.FileEditorManager;
 import com.intellij.openapi.project.Project;
 import com.intellij.openapi.vfs.VirtualFile;
+import com.sourcegraph.project.RepoInfo;
+import com.sourcegraph.util.SourcegraphUtil;
 
+<<<<<<< HEAD:src/main/java/FileAction.java
```

Comment:
@madppiper I don't think you meant to commit these lines. Could you finish the merge before committing?